src/services/dashboards/dashboard-detail/DashboardDetailPage.vue

```diff
@@ -119,7 +121,12 @@ const handleVisibleCloneModal = () => {
 
 // INIT
 (async () => {
-    SpaceConnector.clientV2.dashboard.projectDashboard.get({ project_dashboard_id: 'project-dash-5f7dc1d3bb43' });
+    try {
+        const result: DashboardDetailDataModel = await SpaceConnector.clientV2.dashboard.projectDashboard.get({ project_dashboard_id: 'project-dash-eb13f8b042f9' });
+        state.labelList = result.labels;
+    } catch (e) {
+        ErrorHandler.handleError(e);
+    }
```

Comment:
question: Is this temporary code? Even so, it would be nice to separate them into functions.

Reply:
Yes it is. I'll separate it soon!